cmd/addon-manager/Dockerfile

```diff
@@ -5,9 +5,6 @@ WORKDIR /workspace
 # Copy the Go Modules manifests
 COPY go.mod go.mod
 COPY go.sum go.sum
-# cache deps before building and copying source so that we don't need to re-download as much
-# and so that source changes don't invalidate our downloaded layer
-RUN go mod download
```

Comment:
Is there any detail reason for removing this line?

Reply:
it's for minimizing the container layer

Reply:
OK